src/styles/theme-manager.js

```diff
@@ -167,6 +167,8 @@ export default {
         textColor: rawTheme.palette.accent3Color,
         accentColor: rawTheme.palette.primary1Color,
         clockColor: rawTheme.palette.textColor,
+        clockCircleColor: ColorManipulator.fade(rawTheme.palette.textColor, 0.12),
```

Comment:
Yes, we can add a `clockCircleColor` property. For the dark theme, we should have `clockCircleColor = pickerHeaderColor`